libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -587,7 +587,13 @@ void OverrideChecker::checkOverride(OverrideProxy const& _overriding, OverridePr
 		// This is only relevant for a function overriding a function.
 		if (_overriding.isFunction())
 		{
-			if (_overriding.stateMutability() != _super.stateMutability())
+			// Stricter mutability is always okay except when super is Payable
```

Comment:
It's also not ok when derived is `payable`, isn't it?

Reply:
well, when derived is `payable` it is not stricter but less (or equal) strict so that is implicitly included in that phrasing..